PlanningPoker.App/ViewModels/JoinCommand.cs

```diff
@@ -0,0 +1,60 @@
+namespace PlanningPoker.App.ViewModels
+{
+    using System;
+    using System.Diagnostics;
+    using System.Threading.Tasks;
+    using System.Windows.Input;
+    using PlanningPoker.App.Models;
+    using PlanningPoker.Shared;
+
+    public class JoinCommand
```

Comment:
Den skal helt sikkert hedder JoinViewModel istedet for

Reply:
Nej
For det er ikke en ViewModel?